scripts/mkimage

```diff
@@ -71,8 +71,8 @@ trap cleanup SIGINT
   UUID_2=$(date '+%M%S')
   FAT_SERIAL_NUMBER="${UUID_1}${UUID_2}"
   UUID_SYSTEM="${UUID_1}-${UUID_2}"
-  FAT_VOLUME_LABEL="LIBREELEC"
-  STORAGE_VOLUME_LABEL="STORAGE"
+  FAT_VOLUME_LABEL="${DISTRO_BOOTLABEL}"
```

Comment:
You can simplify mkimage by changing instances of `$FAT_VOLUME_LABLE` to be `$DISTRO_BOOTLABEL` and `$STORAGE_VOLUME_LABEL` to `$DISTRO_DISKLABEL`